sys/net/bpf.c

```diff
@@ -125,7 +125,7 @@ struct bpf_program_buffer {
 #ifdef BPF_JITTER
 	bpf_jit_filter		*func;
 #endif
-	void			*buffer[];
+	void			*buffer[0] __no_subobject_bounds;
```

Comment:
Maybe keep it as a flexible array and just add the annotation?

Reply:
It looks like that to pacify the assertion it needs to have the size, but I need to investigate why

Reply:
moved these commits to another PR.